### ISSUE TITLE
Unable to use Vuesax

### ISSUE THREAD

**Issue by gijo-varghese** [lusaxweb/vuesax#56]
OS: Ubuntu 18.04
NodeJS version: 10
Vuesax version: 3.0.12
Browser version: Chrome 66
Package manager: npm

After installing and using vuesax everything stopped rendering. And its showing an error in console
```
{colorprueba: "240, 91, 28"}
vuesax.common.js?a7ba:6125 Uncaught TypeError: Cannot read property 'hasOwnProperty' of undefined
    at Object.install (vuesax.common.js?a7ba:6125)
    at Function.Vue.use (vue.runtime.esm.js?ff9b:4747)
    at eval (main.js?1c90:14)
    at Object../src/main.js (app.js:1979)
    at __webpack_require__ (app.js:679)
    at fn (app.js:89)
    at Object.0 (app.js:2012)
    at __webpack_require__ (app.js:679)
    at app.js:725
    at app.js:728
```

**Comment by gijo-varghese** [comment]
ok. I've figured out the issue. This error is thrown if no options are set
eg: `Vue.use(Vuesax);`

It works fine if I pass an empty options
`Vue.use(Vuesax, {});`

![screenshot from 2018-05-14 18-25-09](https://user-images.githubusercontent.com/23092846/39998589-283c9e12-57a4-11e8-8612-421ee4b4f443.png)

**Comment by luisDanielRoviraContreras** [comment]
@gijo-varghese Ready and the error is corrected in v3.0.13 thank you very much for the contribution
